README.md

```diff
@@ -1,43 +1,419 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
+# Google Cloud SDK Docker
+
+
+The [Cloud SDK](https://cloud.google.com/sdk/) is the command-line interface to the Google Cloud Platform.  It is a flexible utility which uses GCPs own Cloud APIs to perform many different tasks such as deploying code on AppEngine, to creating Compute Engine VMs to checking IAM permissions and so on.  It is the command line interface to pretty much every Google Cloud Platform API and service.   
+
+This article describes some more ways customers can use it by demonstrating several use cases involving containerizing the Cloud SDK.   The following shows Dockerfiles targeted towards various operating systems and some sample usecases (Its nothing new, just a sample set to document the baseline dockerfiles.
+
+* [Dockerfile](#dockerfile)
+    - [Ubuntu](#ubuntu)
+    - [CentOS](#centos)
+    - [Alpine](#alpine)
+* [Usecases](#usecases)
+    - [Containerize local development environment](#containerize-local-development-environment)
+    - [Run gcloud cli without installing SDK locally](#run-gcloud-cli-without-installing-sdk-locally)
+    - [Run emulators in containers](#run-emulators-in-containers)
+    - [Automate simple DevOps tasklets](#automate-simple-devdps-tasklets)       
+
+## Dockerfile
```

Comment:
i remove the inline dockerfile in the readme.

the part about 'extending' the image is more relevant to a section below where it describe how you can install the additional components, specifically if you need to install appengine-java, then you need to apk install openjdk. (Pls see comment line below)